src/Microsoft.AspNetCore.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -341,7 +358,59 @@ private void BuildFallbackBlock(TagHelperContent builder)
                 .AppendHtml("\",");
 
             AppendFallbackHrefs(builder, fallbackHrefs);
-            builder.AppendHtml("</script>");
+
+            builder.AppendHtml(", \"");
+
+            // Perf: Avoid allocating enumerator
+            for (var i = 0; i < attributes.Count; i++)
+            {
+                var attribute = attributes[i];
+                if (string.Equals(attribute.Name, HrefAttributeName, StringComparison.OrdinalIgnoreCase))
+                {
+                    continue;
+                }
+
+                attribute.WriteTo(StringWriter, HtmlEncoder);
+                StringWriter.Write(' ');
+            }
+
+            var stringBuilder = StringWriter.GetStringBuilder();
+            var scriptTags = stringBuilder.ToString();
+            stringBuilder.Clear();
+            var encodedScriptTags = JavaScriptEncoder.Encode(scriptTags);
+            builder.AppendHtml(encodedScriptTags);
+
+            builder.AppendHtml("\");</script>");
+        }
+
+        private bool HasRelStylesheet(TagHelperAttributeList attributes)
```

Comment:
This name is confusing because the `<link>` doesn't have any kind of style sheet; suggest `HasStyleSheetLinkType()`. (Side note: HTML 5.0 attempted to squash "style sheet" into one word but everyone else, even CSS, disagrees.)